gradle/dependencies.gradle

```diff
@@ -7,6 +7,10 @@ repositories {
         name "JourneyMap"
         url "https://jm.gserv.me/repository/maven-public/"
     }
+    maven {
+        name "Dynmap"
+        url = "https://repo.mikeprimm.com/"
+    }
```

Comment:
Quoth the properties file,
> Dynmap doesn't have a maven

Reply:
I'll edit that comment a bit to state explicitly they do not have release artifacts stored on Maven, only API artifacts